### PR TITLE
chore: [FRONT-95] Card Button color change

### PR DESCRIPTION
**Pull request by umrzoq-toshkentov**
**PR description:**

**Implemented:**

Card Button color change

**Checklist:**

- [x] I ran this code locally
- [x] I wrote the necessary tests
- [x] My code follows the [style guidelines](http://bit.ly/sd-web-style-guide)
- [x] I followed the [instructions](http://bit.ly/sd-web-pr) to create a pull request

**JIRA card:**

https://superdispatch.atlassian.net/browse/FRONT-95

**Should know about:**

image

### CODE REVIEW

File: packages/ui/src/card-button/CardButton.tsx [hunk 1]
```diff
@@ -109,6 +111,20 @@ const useStyles = makeStyles<
       marginRight: theme.spacing(-0.5),
     },
 
+    errorIcon: {
+      marginRight: theme.spacing(1),
+      marginLeft: theme.spacing(-0.5),
+      '& svg': {
+        fill: Color.Red300,
+      },
```
Comment (timurhamitov) [inline packages/ui/src/card-button/CardButton.tsx:119]: @umrzoq-toshkentov Can we set any icon as an error icon? Will `fill` work with any of them properly?

Comment (umrzoq-toshkentov) [reply packages/ui/src/card-button/CardButton.tsx:119]: yes we can set any icon. That is why I just targeted to its svg, which means svg has both material ui component and our custom icons as well.

Comment (timurhamitov) [reply packages/ui/src/card-button/CardButton.tsx:119]: @umrzoq-toshkentov How [alerts](https://ui.superdispatch.org/?path=/docs/lab-alert--basic) are implemented? I just think maybe we do not need to set color for icon here.
If we want a default icon for the error state, maybe we can add it directly into the component with the ability to replace it or disable it?

Comment (umrzoq-toshkentov) [reply packages/ui/src/card-button/CardButton.tsx]: Removed errorIcon and unnecessary classNames

Comment (timurhamitov) [reply packages/ui/src/card-button/CardButton.tsx]: @umrzoq-toshkentov Does the design team know that there will not be an icon by default?

Comment (umrzoq-toshkentov) [reply packages/ui/src/card-button/CardButton.tsx]: @timurhamitov Yes I have replied it in new-color-palette threads, you can check it

**Review by timurhamitov — APPROVED** [review]

**Pull request merged**